flink-jepsen/src/jepsen/flink/db.clj

```diff
@@ -303,7 +303,7 @@
     "-Djobmanager.rpc.address=$(hostname -f)"
     "-Djobmanager.rpc.port=6123"
     "-Dmesos.resourcemanager.tasks.cpus=1"
-    "-Dmesos.resourcemanager.tasks.mem=2048" ;; FLINK-15082: this option must be set instead of taskmanager.memory.process.size
+    "-Dtaskmanager.memory.process.size=2048m" ;; FLINK-15082: this option must be set instead of taskmanager.memory.process.size
```

Comment:
I guess this note could be removed after this PR.

Reply:
Let me fix it